### ISSUE TITLE
Add a Rocket `tera` template example

### ISSUE THREAD

**Issue by Luni-4** [shuttle-hq/shuttle-examples#92]
It would be helpful to add this [tera](https://github.com/SergioBenitez/Rocket/tree/master/examples/templating/templates/tera) example for Rocket to `shuttle-examples`. It only requires to enable the `tera` features in `rocket_dyn_templates`. Is this feature going to work on `shuttle` if enabled?

**Comment by oddgrd** [comment]
Hey! Yes, it should work, but I'm not sure we need a whole new example for it, since Rocket already has an example for it.

**Comment by Luni-4** [comment]
Yes, you're right. Just wondering though whether an example in `shuttle` too could help in implementing a `tera`-template project more straightforward

**Comment by Luni-4** [comment]
The current `shuttle` example could be changed as the [Rocket](https://github.com/SergioBenitez/Rocket/tree/master/examples/templating) example. So it is not necessary to add new examples and the maintenance burden will be reduced. The `Rocket `example uses both the templating mechanisms, so I presume a copy-paste approach should be sufficient, but I do not know the `Shuttle` effort, so I could be wrong

**Comment by ArthurGrizzle** [comment]
Thank you
